packages/lesswrong/components/posts/PostsSingle.jsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 
 const PostsSingle = ({params, router}, context) => {
   const version = router.location && router.location.query && router.location.query.revision
-  return <Components.PostsPage documentId={params._id} version={version} />
+  return <Components.PostsPage documentId={params._id} sequenceId={null} version={version} />
```

Comment:
Why is this necessary?

Reply:
`sequenceId` is context used to determine which sequence's navigation to show on the top/bottom (it needs to be specified separately from the post, since the same post can be in more than one sequence). This component corresponds to a route where no sequence was specified, hence null (the resolver receives the null and fills in the canonical sequence if appropriate). `PostsPage` uses `extraVariables` with the `withDocument` HoC, which (empirically) requires all `extraVariables` to be present; omitting this parameter wouldn't make an implicit null, it would just crash.